src/app/components/import/ImportingModalContent.js

```diff
@@ -119,11 +134,14 @@ const ImportingModalContent = ({ isVcf, file = '', vcardContacts, privateKey, on
 
             const indexMap = contacts.map(({ index }) => index);
 
+            const INFINITE_TIME = 1000 * 60 * 60; // 1 hour
```

Comment:
It should be define at the top of this file.

Reply:
Haha, ok. I wanted to be much more poetic here :grin: